### ISSUE TITLE
Bug: Weird Comma after GE’s (WRT, D, ) remove this comma 

### ISSUE THREAD

**Issue by WinbertZhang** [data-science-ucsb/gauchocourses#31]
<img width="317" alt="Screenshot 2023-01-25 at 8 39 01 PM" src="https://user-images.githubusercontent.com/17103790/214759745-75933f31-886c-473d-b578-b4bf33b03524.png">

**Comment by DanLeEpicMan** [comment]
The bug comes from [this line](https://github.com/data-science-ucsb/gauchocourses/blob/38c19ba38fa9c37c6975b2b1abbeb83a3bef3ba7/frontend/src/components/events/ListItems/CourseListItem.vue#L14).

I attempted a solution where I made a JavaScript method and passed the "ges" parameter into it, in order to let it create a suitable string [(see code here)](https://github.com/data-science-ucsb/gauchocourses/blob/5ff1c4e421ddd1ff5b5de10b2852598af79746e4/frontend/src/components/events/ListItems/CourseListItem.vue#L90-L97). However, this always causes "None" to be displayed, as if the class had no GEs (as if this [if-statement](https://github.com/data-science-ucsb/gauchocourses/blob/5ff1c4e421ddd1ff5b5de10b2852598af79746e4/frontend/src/components/events/ListItems/CourseListItem.vue#L12) failed).

If I let the method return simple strings without actually using the `ges` parameter though, something like
```js
displayGEs: function(ges) {
      return "hey";
    }
```
Then it works fine. Moreover, using GE's explicitly in place of the function call also works fine. This leads me to believe that the issue is reading and modifying the passed object within the `displayGEs` method. I'm not too sure where to proceed from here; all related Vue information I found online wasn't too helpful, but I'll keep my eye out.

**Comment by Ponmo** [comment]
Hey Daniel, I looked over this problem.

So, I saw that each iteration added a comma. So, I wanted to make it so the last iteration doesn't, which means checking with "v-if" to see if the current "ge" was the last element in "ges". Else no comma. The below code should work to fix the problem!

Note: I added a template element (which is basically just vue's element that exists purely for logic) in order to wrap the if-else statement inside the for loop.

<template v-for="ge in ges.sort()">
            <span v-if="ge!=ges.sort().at(-1)" v-bind:key="ge.geCode">{{ ge['geCode'].trim() }}, </span>
            <span v-else v-bind:key="ge.geCode">{{ ge['geCode'].trim() }}</span>
</template>
